### PR TITLE
2.7 fixes

### PR DESCRIPTION
**Pull request by eemurphy**
**Related Issue:**  https://issues.redhat.com/browse/ACM-6005

**Description of Changes:** required quick fix for mce naming conventions

### CODE REVIEW

**Comment by rh-dluong** [comment]
@eemurphy I've reviewed the PR and nothing was done about the review,  is there a reason I'm being pinged? Action is not on me for it going stale.

File: collection-scripts/mce-gather.sh [hunk 1]
```diff
@@ -80,14 +80,14 @@ gather_spoke () {
...
-    oc get csv -n "${OPERATOR_NAMESPACE}" > ${BASE_COLLECTION_PATH}/gather-acm.log
+    oc get pods -n "${OPERATOR_NAMESPACE}" > ${BASE_COLLECTION_PATH}/gather-mce.log
+    oc get pods -n "${DEPLOYMENT_NAMESPACE}" > ${BASE_COLLECTION_PATH}/gather-mce.log
+    oc get csv -n "${OPERATOR_NAMESPACE}" > ${BASE_COLLECTION_PATH}/gather-mce.log
     oc adm inspect  ns/"${DEPLOYMENT_NAMESPACE}"  --dest-dir=must-gather
     oc adm inspect  ns/"${OPERATOR_NAMESPACE}"  --dest-dir=must-gather
     oc adm inspect  ns/open-cluster-management-hub  --dest-dir=must-gather
     # request from https://bugzilla.redhat.com/show_bug.cgi?id=1853485
-    oc get proxy -o yaml > ${BASE_COLLECTION_PATH}/gather-proxy.log
+    oc get proxy -o yaml > ${BASE_COLLECTION_PATH}/gather-proxy-mce.log
```
[the remark this answers is not in the document]
Comment (eemurphy) [reply collection-scripts/mce-gather.sh]: @rh-dluong fixes have been made, apologies for the delay!!

**Pull request merged**